test/jdk/javax/xml/jaxp/datatype/8033980/SerializationTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2014, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
missed the original copyright year, e.g. 2014, 2023,

Reply:
Added the copyright year and added the JDK version information in Data files.